packages/demux/src/DbUpdater.ts

```diff
@@ -1,56 +1,75 @@
-import { ActionCallback, Updater } from 'demux';
-
+import type { ActionCallback, BlockInfo, Updater } from 'demux';
 import type { EntityConfig } from '@aloxide/bridge';
 import type { DataAdapter } from './DataAdapter';
 import type { Logger } from './Logger';
+import type { AloxideActionHandlerContext } from './AloxideActionHandler';
 
-interface DbUpdaterOptions<ID, D> {
+export interface DbUpdaterOptions<ID, D> {
   actionType: string;
   entity: EntityConfig;
   dataAdapter: DataAdapter<ID, D>;
   logger?: Logger;
 }
 
+export interface DbUpdaterActionPayload {
+  data: any;
+  [key: string]: any;
+}
+
 /**
  * An implementation of updater which uses DataAdaper to abstract the
  * database layer
  */
 export class DbUpdater<ID, D> implements Updater {
   entity: EntityConfig;
-  dataAdaper: DataAdapter<ID, D>;
+  actionType: string;
+  dataAdapter: DataAdapter<ID, D>;
   logger?: Logger;
   revert?: ActionCallback;
```

Comment:
Field `revert` is not implemented yet. Need research to write a document about it and implement it if needed.

Reply:
continue in #78